frontend/src/components/placementVisuals/Placement.tsx

```diff
@@ -90,14 +90,9 @@ function Placement({ stepID, sampleIDs }: PlacementProps) {
       if (!cells) return false
       const [Rows = [] as const, Columns = [] as const] = activeDestinationContainer.spec
       const containerSize = Rows.length * Columns.length
-      let placedCells = 0
-      for (const destinationCell of cells) {
-        if (destinationCell.placedFrom) {
-          placedCells++
-        }
-      }
+      const placedCells = cells.reduce((acc, cur) => {return cur.placedFrom ? ++acc : acc}, 0)
```

Comment:
Eheh. You didn't have to, but at least you gained some experience reducing ;)
The lambda could simply be `(acc, cur) => cur.placedFrom ? acc + 1 : acc` ;)